wvlet-ui-main/src/main/scala/wvlet/lang/ui/editor/WvletMonarchLanguage.scala

```diff
@@ -138,61 +139,124 @@ val WvletMonarchLanguage: IMonarchLanguage =
 
     val symbols = "[=><!~?:;,\\._@$&|+\\-*/^%]+"
 
+    val numberTokenRules: Rules = js.Array(
+      js.Array(
+        js.RegExp("[1-9][0-9_]*.[0-9]+"),
+        new:
+          val token = "number.float"
+      ),
+      js.Array(
+        js.RegExp("0[xX][0-9a-fA-F_]+"),
+        new:
+          val token = "number"
+      ),
+      js.Array(
+        js.RegExp("[1-9][0-9_]*"),
+        new:
+          val token = "number"
+      )
+    )
+
+    val keywordTokenRules: Rules = js.Array(
+      js.Array(
+        js.RegExp("[a-z_][a-z_0-9\\.]*"),
+        new:
+          val cases = js.Dictionary(
+            "@keywords"     -> "keyword",
+            "@typeKeywords" -> "type.keyword",
+            "@default"      -> "identifier"
+          )
+      )
+    )
+
+    type Rules = js.Array[js.Array[js.Object]]
```

Comment:
Without this type annotation, `js.RegExp` and the rule object cannot be in the same array. To avoid too much repetition, declared type.